proxy/api/src/http/identity.rs

```diff
@@ -47,6 +48,18 @@ fn get_filter(
         .and_then(handler::get)
 }
 
+/// `GET /remote/<id>`
```

Comment:
I wonder if we need a separate endpoint for remote identities. We could probably have a "type" field on `Identity` to distinguish whether it's our own identity or a remote one. But I understand that this was the easiest way, so maybe we can refactor this later in a follow-up.

What do you think @geigerzaehler?

Reply:
It think it makes sense to have separate endpoints because we also have different fields for the two types of identities.